snippets/ztp-performance-profile.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: SNIPPET
+.Recommended performance profile configuration
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: perfprofile-policy
+spec:
+  additionalKernelArgs:
+    - idle=poll
+    - rcupdate.rcu_normal_after_boot=0
+  cpu:
+    isolated: 2-19,22-39 <1>
```

Comment:
2-51,54-103

Reply:
In `out/source-crs/PerformanceProfile.yaml` I see 
```yaml
    pages:
      - size: $size
        count: $count
        node: $node
```

Do we need node, and what would the $node value be in a manual CR?

Reply:
Right. I don't see node on my sno.